WebCoffeeMachine/coffeemachine/static/coffeemachine/style.css

```diff
@@ -193,4 +193,32 @@ h3 {
 
 .detail-block-1, .detail-block-2 {
 	display: block;
-}
\ No newline at end of file
+}
+
+#container {
+	opacity: 1;
+}
+
+#overlay {
+	position: fixed;
+	width: inherit;
+	height: inherit;
+	position: absolute;
+	top: 35%;
+	left: 45%;
+	display: none;
+	color: white;
+	font-family: "Times New Roman", Helvetica, sans-serif;
+}
+
+#overlay img {
+	width: 50%;
+	height: 50%;
+}
+
+#overlay { animation: fadeIn 1s; }
```

Comment:
i would put overlay before overlay img

Reply:
The adjustment made in commit # [e2bf76a](https://github.com/EsamSharaf/WebCoffeeMachine/pull/5/commits/e2bf76aebbff2f2efcba382ba26999c19d7dfa4c)